### PR TITLE
fix: ci for dev server next

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

fix

**Did you add tests for your changes?**
wip

**If relevant, did you update the documentation?**
NA

**Summary**
Fix CI for dev server next

**Does this PR introduce a breaking change?**
no

**Other information**

### CODE REVIEW

**Comment by snitin315** [comment]
> /cc @snitin315 Can you help and update negative flags in webpack-dev-server? Some of them lost description, look at CI

On it.

**Comment by snitin315** [comment]
@alexander-akait  we need to add support for `negatedDescription` in `webpack.cli.getArguments()`, I will send a PR.

**Comment by alexander-akait** [comment]
Yep let's do it

**Review by alexander-akait — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Comment by alexander-akait** [comment]
We have bug with multi compiler mode, I will fix it in other PR

File: packages/serve/src/index.ts [hunk 1]
```diff
@@ -174,7 +174,8 @@ class ServeCommand {
                         }
                         return accumulator;
                     }, {});
-                    const result = Object.assign({}, compiler.options.devServer);
+                    // TODO fix compatibility with multi compiler mode
+                    const result = Object.assign({}, (compiler.options || {}).devServer);
```
Comment (evenstensberg) [inline packages/serve/src/index.ts:178]: Another line with the assign object would be better

Comment (alexander-akait) [reply packages/serve/src/index.ts:178]: it is workaround for broken logic, as I described above it will be fixed in other PR

**Comment by alexander-akait** [comment]
hate macos... don't understand why it break buffer, will be great if somebody try to fix it locally

**Comment by anshumanv** [comment]
@alexander-akait I think since the options increased it's not able to display the full list in MacOS because it's clipped, for now I think we should skip those tests for macOS

**Comment by alexander-akait** [comment]
We should report about this in node.js, it is broken behavior

**Pull request merged**